src/light.js

```diff
@@ -203,7 +221,25 @@ class DirectionalLight extends Light {
         this.type = 'directional';
         this.struct_name = 'DirectionalLight';
 
-        this.direction = (config.direction || [0.2, 0.7, -0.5]).map(parseFloat); // [x, y, z]
+        if (config.direction) {
+            this.direction = config.direction;
+        }
+        else {
+            // Default directional light maintains full intensity on ground, with basic extrusion shading
+            let theta = 135; // angle of light in xy plane (rotated around z axis)
+            let scale = Math.sin(Math.PI*60/180); // scaling factor to keep total directional intensity to 0.5
```

Comment:
Ah I see, yeah tangram-es normalizes these CPU-side so I wanted to make sure the results wouldn't differ.